conference/src/main/java/com/kyriba/conference/management/api/ScheduleController.java

```diff
@@ -30,78 +37,74 @@
 @RequestMapping("/api/v1/schedule")
 public class ScheduleController
 {
+  private final ScheduleService scheduleService;
+
+
+  @Autowired
+  public ScheduleController(ScheduleService scheduleService)
```

Comment:
Could be simplified with AllArgsConstructor annotation

Reply:
done